src/components/output/OutputGesture.svelte

```diff
@@ -29,11 +29,8 @@
   }
   const wasTurnedOff = () => {};
   const wasTurnedOn = () => {
-    if (Microbits.isOutputMakecode()) {
-      ConnectionBehaviours.getOutputBehaviour().onGestureRecognized(
-        $gesture.ID,
-        $gesture.name,
-      );
+    if ($state.outputOrigin === HexOrigin.MAKECODE) {
+      Microbits.getOutputMicrobit().sendToOutputUart('g', $gesture.name);
```

Comment:
At some point it'd be good to replace sendToOutputUart with more meaningful methods.

We should also consider whether we've made this more fragile in the case the output micro:bit goes away.
Doesn't matter for our current UI but worth pondering. Could make getOutputMicrobit() not throw.

Reply:
We did this.